internal/agent/device/console/console.go

```diff
@@ -141,6 +141,10 @@ func (c *ConsoleController) startForwarding(ctx context.Context, stdin io.WriteC
 				c.log.Errorf("stream > bash:error receiving message for stdin: %s", err)
 				return fmt.Errorf("stream > bash:error receiving message for stdin: %w", err)
 			}
+			if msg == nil {
```

Comment:
can this happen in practice? my understanding is the only way Recv can return a nil value is if err is non nil.

Reply:
This defensive writing. The logic will panic without it for a nil message.
Such case is considered as an error.
I can adjust the test case and send a non-nil message.

If this can happen, I prefer a graceful error instead of an exception.

Reply:
Dropped this check